cmd/ocp.go

```diff
@@ -63,6 +63,9 @@ func openShiftCmd() *cobra.Command {
 	ocpCmd.MarkFlagsRequiredTogether("es-server", "es-index")
 	ocpCmd.MarkFlagsMutuallyExclusive("es-server", "local-indexing")
 	ocpCmd.PersistentPreRun = func(cmd *cobra.Command, args []string) {
+		if cmd.Name() == "version" {
```

Comment:
I am bit confused on where this issue is actually happening. When I try to reproduce in my local, I do not see any issues.
```
[vchalla@vchalla-thinkpadp1gen2 kube-burner-ocp]$ kube-burner-ocp version
Version: main
Git Commit: dc8ff253f51b379fb63fe703e263ffc3e184e1d6
Build Date: 2024-02-14-09:50:57
Go Version: go1.21.5
OS/Arch: linux amd64
```

Reply:
The issue is that when you run `kube-burner-ocp version` it tries to discover the prometheus endpoint + token. For example, it fails when kubeconfig is not configured